gstd/src/common/handlers.rs

```diff
@@ -25,50 +25,208 @@
 //! debug and non-debug mode, for programs built in `wasm32` architecture.
 //! For `debug` mode it provides more extensive logging.
 
+#[cfg(target_arch = "wasm32")]
 #[cfg(feature = "oom-handler")]
 #[alloc_error_handler]
 pub fn oom(_: core::alloc::Layout) -> ! {
     crate::ext::oom_panic()
 }
 
-#[cfg(feature = "panic-handler")]
-mod panic_handler {
-    use crate::ext;
-    use core::panic::PanicInfo;
+/// We currently support 3 panic handler profiles:
+/// - `panic-handler`: it displays `no info`
+/// - `panic-message`: it displays `'{message}'`
+///   - In nightly Rust, we use `#![feature(panic_info_message)]` and the
+///     [`write!`] macro.
+///   - In stable Rust, we need to modify the default panic handler message
+///     format.
```

Comment:
move this below

Reply:
this is due to doc tests
`panic_handler::internal` - wasm thing, `panic_handler::` - doc tests

Reply:
ed7741062e067677412d4bd8f6c7103251f718ab